setup.py

```diff
@@ -0,0 +1,43 @@
+"""
+Setuptools based setup module
+"""
+from setuptools import setup, find_packages
+
+
+setup(
+    name='pyiron_contrib',
+    version='0.0.1',
+    description='Repository for the pyiron plugins developed at the Max Planck Institute für Eisenforschung.',
```

Comment:
Rather 'Repository for user-generated plugins to the pyiron IDE developed by the Max Planck Institute für Eisenforschung'.

I guess the most widely used plugin (in terms of geography: Austria+Canada) is Dominik's QM/MM protocol, and he's not an MPIE memeber.

Reply:
I copied the file from `pyiron_mpie`, now I phrased it `Repository for user-generated plugins to the pyiron IDE`